bigtable-hbase-dataflow/src/main/java/com/google/cloud/bigtable/dataflow/HBaseMutationCoder.java

```diff
@@ -27,14 +27,14 @@
 import org.apache.hadoop.hbase.protobuf.generated.ClientProtos.MutationProto;
 import org.apache.hadoop.hbase.protobuf.generated.ClientProtos.MutationProto.MutationType;
 
-import com.google.api.services.dataflow.model.Sink;
 import com.google.cloud.dataflow.sdk.Pipeline;
 import com.google.cloud.dataflow.sdk.coders.AtomicCoder;
 import com.google.cloud.dataflow.sdk.coders.Coder;
 import com.google.cloud.dataflow.sdk.coders.CoderException;
 
 /**
- * A {@link Coder} that serializes {@link Mutation} objects used by the Cloud Bigtable {@link Sink}.
+ * A {@link Coder} that serializes and deserializes the HBase {@link Mutation} objects
+ * using {@link ProtobufUtil}.
```

Comment:
I am OK with not mentioning ProtoBufUtil. @hegemonic  Are you OK with putting more context as Solomon suggested? A potential problem with the context could be that it could be changed later, and what are are saying here could be outdated later.

"When Dataflow notices a slowdown in executing Puts and Deletes, it will send those Puts and Deletes to be processed on another Dataflow machine. This class handles the serialization and desrialization of the Puts and Deletes."